pages/signup/finalize.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { FinalizePage } from 'views/FinalizePage'
+
+function Finalize() {
+  return (
+    <FinalizePage />
```

Comment:
1. It should be other way around. FinalizaPage should contain Finalize component, not vice versa.
2. Also it would be simpler if you wrote this as functional component like so:
`const FinalizePage = () => <Finalize />`
instead of 5/6 Lines Of Code, especially when you don't do any extra operations before return statement.